addons/web/static/src/core/dialog/dialog.scss

```diff
@@ -1,7 +1,7 @@
 .modal.o_technical_modal .modal-footer {
     // FIXME: These selectors should not be necessary if we used buttons
     // as direct children of the modal-footer as normally required
-    div,
+    div:not(.o_field_widget),
```

Comment:
I suggest to remove this line because I think this rule was sibling the direct div child and it seems that we doesn't have it anymore. So, I also check the comment just above and I only see 7 potential problematic cases. For the footer selector, the gap seems useful.

Reply:
I tested theses cases and it seems fine.